web/workflowApi.go

```diff
@@ -95,7 +103,13 @@ func (wf *workflowAPI) createWorkflow(w http.ResponseWriter, r *http.Request) {
 		apiMessageResponse(w, http.StatusInternalServerError, "internal server error")
 		return
 	}
-	workflow, err := services.CreateWorkflow(createBody.TopicName, createBody.FunctionName)
+	parsedURL, err := url.ParseRequestURI(createBody.SinkURL)
```

Comment:
I don't understand what this check is for? Why do we want to check the scheme and the host?

Reply:
Yeah you're right, I may have used to mess around with localhost, then forgot to delete. 
It's best to convert it to disposable variable.